src/main.rs

```diff
@@ -28,24 +28,24 @@ fn error(err: String) -> io::Error {
 
 #[tokio::main]
 async fn run_client() -> anyhow::Result<()> {
-    let url = match env::args().nth(1) {
+    let url = match env::args().nth(2) {
```

Comment:
Could you please explain why is it nth(2)?
nth(0) is the executable name, nth(1) is first argument, etc.

Reply:
I just ran "cargo run client url subj pin", is it wrong?

Reply:
"client" is not needed :)
so you run it either as "cargo run url subj pin" or ".\target\debug\client.exe url subj pin"

Reply:
Got it! Thank you!